docs/nitclk/installation.inc

```diff
@@ -0,0 +1,18 @@
+
+.. _nitclk_installation-section:
+
+Installation
+------------
+
+As a prerequisite to using the nitclk module, you must install the NI-TClk runtime on your system. Visit `ni.com/downloads <http://www.ni.com/downloads/>`_ to download the driver runtime for your devices.
```

Comment:
I don't think this is accurate. I think NI-TClk is installed for you when you install the runtime for a driver that supports it.

Reply:
I imagine that drivers that have NI-TClk property `nitclk.SessionReference` will need to install the `nitclk` module too. We can revisit all this then.